cmd/bad-key-revoker/main.go

```diff
@@ -357,10 +357,16 @@ func main() {
 			// keyHashToSerial table at once
 			FindCertificatesBatchSize int
 
-			// Interval specifies how long bad-key-revoker should sleep between attempting to find
-			// blockedKeys rows to process when there is no work to do
+			// Interval specifies the minimum duration bad-key-revoker
+			// should sleep between attempting to find blockedKeys rows to
+			// process when there is no work to do.
 			Interval cmd.ConfigDuration
 
+			// MaxBackoff specifies a maximum amount of time the
+			// backoff algorithm will wait before retrying in the event of
+			// error or no work to do.
+			MaxBackoff cmd.ConfigDuration
```

Comment:
The bad-key-revoker already has a maximumRevocations config. I thought to keep it the same with MaximumBackoff, but am much more in favor of using max in most places because it is a ubiquitously accepted, shorter, variation. In this case, I think starting it with either MaxBackoff or MaximumBackoff would be more in line with the other config values. I could be convinced of MaximumBackoff if only because the longer version is already in use.

Reply:
I don't think there are enough examples either way (`signFailureBackoffMax` in ocsp-updater, `maximumRevocations` in bad-key-revoker) to make this decision for us. Go with whatever you think is the most legible in this context.

Reply:
Went with BackoffDurationMax. It seems like the most descriptive and fits with the style I went with in the pr of a backoff* prefix for the variables.